extend/common-interface/development-branches.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Development branches
+permalink: /extend/common-interface/development-branches/
+---
+
+* TOC
+{:toc}
+
+Development branches are a feature that handles change management in Keboola Connection projects. To learn more about how development branches work refer to the [user documentation](https://help.keboola.com/change-management/). 
+
+{% include branches-beta-warning.html %}
+
+## Running a component in a branch
+
+A component that uses the [Common Interface](/extend/common-interface/) can be run in a branch without any changes to the code. Notable exceptions are components that modify external resources (for example database writers) and components that use injected Storage token to interact with Storage API. 
+
+### How do you tell component is executed in branch context?
+
+When [runner](/extend/docker-runner/) executes a job in a branch, it sets [`KBC_BRANCHID` environment variable](/extend/common-interface/environment/#environment-variables) to current branch id. This ID is unique for each branch in the whole stack.  
+
+### Input and output mapping in development branch
```

Comment:
Přidat explicitně, že v komponentě nikdy nepracuješ s branched storagem a dělá to runner za tebe. 

(maximálně pokud máš token).